RyanMillerC/better-vim-tmux-resizer
# Create Repeatable Test Environment

* Create repeatable manual testing environment
  * Use basic configuration files from */test*
  * Start up with `make test`
  * Should allow for easier debugging
* Create contributing guide
